docs/ReleaseHistory.md

```diff
@@ -11,6 +11,13 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- BRK: Derived keys and hashed data are no longer supported. The following API are removed:
+  - `IdentifiableSecrets.CommonAnnotatedDerivedKeySignature`
+  - `IdentifiableSecrets.CommonAnnotatedHashedDataSignature`
+  - `IdentifiableSecrets.ComputeDerivedCommonAnnotatedKey`
+  - `IdentifiableSecrets.ComputeCommonAnnotatedHash`
```

Comment:
Oops, I missed `IdentifiableSecrets.ComputeDerivedIdentifiableKey` in this list.